hack/.packages

```diff
@@ -0,0 +1,219 @@
+k8s.io/kops
```

Comment:
https://github.com/kubernetes/kops/pull/10407 removes .packages, probably a rebase is needed. If so, please also squash the commits.

Reply:
Ah I see. Fixed!